packages/block-editor/src/hooks/style.js

```diff
@@ -99,7 +104,10 @@ const skipSerializationPathsEdit = {
 		TYPOGRAPHY_SUPPORT_KEY,
 	],
 	[ `${ SPACING_SUPPORT_KEY }.__experimentalSkipSerialization` ]: [
-		'spacing',
+		DIMENSIONS_SUPPORT_KEY,
+	],
+	[ `${ SPACING_SUPPORT_KEY }.__experimentalSkipSerialization` ]: [
+		SPACING_SUPPORT_KEY,
```

Comment:
I think we might have a tiny copy-and-paste error here, shouldn't the dimensions skip serialization check look at:
```
[ `${ DIMENSIONS_SUPPORT_KEY }.__experimentalSkipSerialization` ]
```

The edits to block.json files to opt into the min-height is under `dimensions`, so I'd expect to be able to add the skip serialization flag under there as per other supports.

Reply:
Oh, good catch! Went too quickly with the copy+paste 😄